js/main.js

```diff
@@ -0,0 +1,23 @@
+const getRndInteger = function (min, max) {
+  const randomNumber = Math.floor(Math.random() * (max - min + 1)) + min;
+
+  if (min < 0 || min >= max || max < 0) {
+    return ('Введены некорректные параметры');
+  }
+
+  return randomNumber;
+};
+
+getRndInteger(1, 10);
+
+const getRndNumber = function (min, max, decimal) {
```

Comment:
Первое название функции указал что целое число возвращается, тогда и здесь стоит 
```suggestion
const getRandomFloatNumber = (min, max, decimal) => {
```
Для количества знаков после запятой можно указать значение по умолчанию. Погугли как это сделать